demos/whisper-base/whisper.js

```diff
@@ -23,19 +23,19 @@ import {
 } from "./utils.js";
 
 let tokenizerPath = "";
-let processerPath = "";
+let processorPath = "";
 if (
     location.href.toLowerCase().indexOf("github.io") > -1 ||
     location.href.toLowerCase().indexOf("huggingface.co") > -1 ||
     location.href.toLowerCase().indexOf("vercel.app") > -1
 ) {
     let path = "microsoft/whisper-base-webnn/resolve/main";
     tokenizerPath = `${path}/tokenizer`;
-    processerPath = `${path}/processer`;
+    processorPath = `${path}/processor`;
```

Comment:
Thanks @NingW101 for figuring this out (processer -> processor), but Intel has no permission or it needs extra Microsoft's process and effort to change the path in https://huggingface.co/microsoft/whisper-base-webnn/tree/main/processer to make it work, can we just keep it as it is right now? Thanks 

CC @fdwr

Reply:
Sure, updated.

Reply:
Ah, typo - good to know.